app/src/components/disclosure/styles.tsx

```diff
@@ -0,0 +1,99 @@
+import { css } from "@emotion/react";
+
+export const disclosureGroupCSS = css`
```

Comment:
Just a thought - that we don't really need a separate file if the styles are not going to be re-used. It might be simpler to colocate the styles with the component. NBD

Reply:
They were distracting me so I moved em out. I will keep it in mind for future components